src/api-client.ts

```diff
@@ -167,11 +173,16 @@ export class RevAiApiClient {
         }
     }
 
-    async getTranscriptText(id: string): Promise<string> {
+    async getTranscriptText(id: string, fileName = null, fileLocation = null): Promise<string> {
         try {
             const response = await axios.get(`/jobs/${id}/transcript`, {
                 headers: { 'Accept': 'text/plain' }
             });
+
+            if (fileName) {
+                const fullPath = path.join(fileLocation || '', `${fileName}.txt`);
+                await this.writeOutputToFile(fullPath, response.data);
```

Comment:
This convenience feature wasn't a part of the Acceptance Criteria for this issue. Are we sure we want to add this feature? We will have to support it as we continue development on the SDK.

Reply:
I think we should add it. It's a very nice feature to have. I meant to add it as an AC